src/app/pages/list/inline-local-actions/inline-local-actions.component.html

```diff
@@ -0,0 +1,216 @@
+<mat-toolbar data-cy="blui-toolbar" color="primary" class="app-bar mat-elevation-z2">
+    <button
+        data-cy="toolbar-menu"
+        style="margin-left: -8px"
+        mat-icon-button
+        (click)="toggleMenu()"
+        *ngIf="isLeftPaneVisible && isSmall"
+    >
+        <mat-icon>menu</mat-icon>
+    </button>
+    <div class="mat-title" [style.marginLeft.px]="isSmall ? 24 : 0" *ngIf="isLeftPaneVisible; else showRightPanelTitle">
+        Local Item Actions
+    </div>
+
+    <ng-template #showRightPanelTitle>
+        <button data-cy="arrow-back" mat-icon-button (click)="backToLeftPanel()" class="arrow-back">
+            <mat-icon>arrow_back</mat-icon>
+        </button>
+        <div class="mat-title" *ngIf="showScoreCard">
+            Battery Service
+        </div>
+        <div class="mat-title" *ngIf="showDeviceEditView">
+            Device
+        </div>
+        <div class="mat-title" *ngIf="showLanguageListView">
+            Language
+        </div>
+    </ng-template>
+</mat-toolbar>
+<blui-slide-panel-inline-actions [activePane]="isLeftPaneVisible ? 'left' : 'right'">
+    <div leftPane>
+        <section class="local-inline-actions-example">
+            <div class="list-example">
+                <mat-accordion class="accordion-panel">
+                    <mat-expansion-panel class="mat-elevation-z1" [expanded]="true">
+                        <mat-expansion-panel-header>
+                            <mat-panel-title>
+                                Today
+                            </mat-panel-title>
+                        </mat-expansion-panel-header>
+                        <mat-list class="device-detail-section">
+                            <blui-info-list-item
```

Comment:
Not a big deal for this PR. Just wondering to myself why there are no ripples on mousedown...

Reply:
Not sure, if we want we can log an issue.